index.js

```diff
@@ -1,72 +1,98 @@
-import { Scene, PerspectiveCamera, WebGLRenderer, Color, TorusKnotGeometry, SphereGeometry, FontLoader, TextBufferGeometry } from 'three';
-import { OrbitControls } from "three/examples/jsm/controls/OrbitControls";
+import * as THREE from 'three';
```

Comment:
I think we prefer explicit imports over importing all under a namespace, ie `import { thing } from "three"` preferred to ` import * as THREE from "three" `

Reply:
Hi, thanks for letting me know this, I've changed it back: 2e9ce944051a93b90541acdd85431cc5d3cb12c2